setup.py

```diff
@@ -58,19 +103,17 @@ def build_extension(self, ext):
                                                               self.distribution.get_version())
         if not os.path.exists(self.build_temp):
             os.makedirs(self.build_temp)
-        
+
         cmake_args += ['-DUSE_PYALEMBIC=On']
-        subprocess.check_call(['cmake', ext.sourcedir] + cmake_args, cwd=self.build_temp, env=env)
-        subprocess.check_call(['cmake', '--build', '.'] + build_args, cwd=self.build_temp)
-	
-        # Copy missing dependencies
-        site_dir = "/usr/local/lib/python2.7/site-packages"
-        missing_packages = ["iexmodule.la", "iexmodule.so", "imathmodule.la", "imathmodule.so", "imathnumpymodule.la", "imathnumpymodule.so"]
-        for package in missing_packages:
-            shutil.copyfile(os.path.join(site_dir, package), os.path.join(extdir, package))
+        subprocess.check_call(
+            [cmake_command, ext.sourcedir] + cmake_args, cwd=self.build_temp, env=env)
+        subprocess.check_call(
+            [cmake_command, '--build', '.'] + build_args, cwd=self.build_temp)
+
+
 setup(
     name='alembic',
-    version='1.7.16',
+    version='1.8.2',
```

Comment:
please add a comment to highlight that this needs to be manually updated to match the currrent abc version!

Reply:
https://github.com/framestore/alembic/blob/919f4f4cb914b5a9936e6d6f315f618c16e56ea3/setup.py#L114-L116

Added comment about versioning.